git_machete/git_operations.py

```diff
@@ -299,6 +302,11 @@ def checkout(self, branch: str) -> None:
         self._run_git("checkout", "--quiet", branch, "--")
         self.flush_caches()
 
+    def checkout_pr_refs(self, pr_number: int, branch: str) -> None:
```

Comment:
"Pull request" is not a term known to vanilla git; it's up to services like github to define.

Reply:
I'd rather expec